test/scripts/git-server-cmds/delete-repo

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# delete a git repo in /home/user/repos/$1.git
+# $1 is the name of the repo
```

Comment:
can these two files be a single file we import and wrap these in functions? just trying to reduce bash sprawl

Reply:
In this case we can't, because this is what the git shell expects, one single binary per git command. Those are the bash scripts that we put on the containers to be found by https://github.com/flightctl/flightctl/pull/279/files#diff-17d3cf62a08f372fc25c4dd5a52e747ba04bca4b89e471a094d76332f556b13bR7 `/usr/bin/git-shell`